README.md

```diff
@@ -0,0 +1,245 @@
+
+### NeoFS network environment variables
+The following variables must be passed as
+[GitHub Actions vars context](https://docs.github.com/en/actions/learn-github-actions/variables#using-the-vars-context-to-access-configuration-variable-values) 
+or [GitHub Actions environment variables](https://docs.github.com/en/actions/learn-github-actions/variables).
+
+Up-to-date information on the NeoFS network can be viewed on https://status.fs.neo.org.
+
+If you are using the NeoFS mainnet, we recommend that you do not change `NEOFS_NETWORK_DOMAIN`
+and `NEOFS_HTTP_GATE` environment variables.
+
+| Key                    | Value                                                                                 | Required | Default                |
+|------------------------|---------------------------------------------------------------------------------------|----------|------------------------|
+| `NEOFS_NETWORK_DOMAIN` | Rpc endpoint domain address                                                           | **No**   | st1.storage.fs.neo.org |
+| `NEOFS_HTTP_GATE`      | HTTP Gateway domain address                                                           | **No**   | http.fs.neo.org        |
+| `STORE_OBJECTS_CID`    | Container ID for your data. For example: 7gHG4HB3BrpFcH9BN3KMZg6hEETx4mFP71nEoNXHFqrv | **Yes**  | N/A                    |
+
+
+### Workflow environment variables
+The following variables must be passed as
+[GitHub Actions vars context](https://docs.github.com/en/actions/learn-github-actions/variables#using-the-vars-context-to-access-configuration-variable-values)
+or [GitHub Actions environment variables](https://docs.github.com/en/actions/learn-github-actions/variables).
+
+| Key                 | Value                                                                                                                                                                                                                                                                      | Required | Default |
+|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------|
+| `PATH_TO_FILES_DIR` | Path to the directory with the files to be pushed                                                                                                                                                                                                                          | **Yes**  | N/A     |
+| `RUN_ID`            | GitHub run ID. The ID will be a unique part of the URL address. If a new file with the same name is uploaded to the same container and with the same ID, the new file will replace the old one. We recommend using a combination of github.run_number, start time, and sha | **Yes**  | N/A     |
```

Comment:
Not sure about replacements and I'd add abstract attributes here, someone may want to have "RUNID", someone may want to have "BUILDID", there are lots of other potential cases as well.

Reply:
> Not sure about replacements and I'd add abstract attributes here, someone may want to have "RUNID", someone may want to have "BUILDID", there are lots of other potential cases as well.

Fixed